acceptance/analyzer_test.go

```diff
@@ -820,6 +789,51 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 				})
 			})
 
+			when("called with previous image", func() {
+				it.Before(func() {
+					h.SkipIf(t, api.MustParse(platformAPI).Compare(api.MustParse("0.7")) < 0, "Platform API < 0.7 does not support -previous-image")
+				})
+
+				when("auth registry", func() {
+					var authRegAppImage, appAuthConfig string
+
+					it.Before(func() {
+						metadata := minifyMetadata(t, filepath.Join("testdata", "analyzer", "app_image_metadata.json"), platform.LayersMetadata{})
+						authRegAppImage, appAuthConfig = buildAuthRegistryImage(
+							t,
+							"some-app-image-"+h.RandString(10),
+							filepath.Join("testdata", "analyzer", "app-image"),
+							"--build-arg", "fromImage="+containerBaseImage,
+							"--build-arg", "metadata="+metadata,
+						)
+					})
+
+					it("writes analyzed.toml with previous image identifier", func() {
```

Comment:
Are there any other tests that we would like to add that relates to the previous image?
Maybe if it's not accessible / doesn't exists?

Reply:
The tests we have in place for `image` should cover that, I believe. Since `previousImage = image`

Reply:
Yeah, I thought about it but since we always passing `image`, I was wondering if we can test it directly, mostly in case something will change in the future.

Reply:
I think this is covered by the new acceptance test that Jesse added.

Reply:
I see what you're saying - internally we know that we assign `-previous-image` to `Analyzer.Image` and so they should be treated equivalently, but it's not obvious from the tests. However since we're passing `"some-fake-image"` as the positional argument and yet getting metadata for the previous image I think we're getting pretty good coverage from this test.

Reply:
And what about passing two "real" images? One for the `image` and one for the `previousImage`?

Reply:
IDK, not sure if there is significant value and it would create more overhead. @jabrown85 WDYT?

Reply:
I'm fine leaving it as-is, but will add more tests if you two think it is valuable enough!

Reply:
Personally I think it's better to create another test so our coverage will be better for future changes, but if you both think it's unnecessary, we can keep it as is for now.

Reply:
🤷🏼‍♀️

Reply:
Added new test in 251ba00.